pylint/checkers/stdlib.py

```diff
@@ -287,6 +317,42 @@ def _check_open_mode(self, node):
                 self.add_message('bad-open-mode', node=node,
                                  args=mode_arg.value)
 
+    def _check_invalid_envvar_value(self, call_arg, infer):
+        if isinstance(call_arg, Call):
```

Comment:
We have an utility called [safe_infer](https://github.com/PyCQA/pylint/blob/56a6723f96dddd1e70e668d808e8446fd53b841f/pylint/checkers/utils.py#L786) which you should use instead.